octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,13 @@
+package org.example.calculator;
+
+public class Calculator {
+
+    public static void main(String[] args) {
+        System.out.println("Sum of firstNumber and secondNumber is: " + add(10, 20));
```

Comment:
[Severity: Minor] The method name 'add' is clear, but consider adding a comment to explain what it does.
Adding a comment above the method can clarify its purpose and usage for anyone reading the code.
Rationale: Clear documentation helps others understand your code better.
Reference: https://www.oracle.com/java/technologies/javase/codeconventions-contents.html